### ISSUE TITLE
Forgot password

### ISSUE THREAD

**Issue by harsh500** [Deepak-Sangle/mIIT-Kute#2]
Hii,
Can you please add a forgot password button as I literally forgot my password :( and now I only have 1 iitk id so there so other way to login.
Harsh

**Comment by Deepak-Sangle** [comment]
Yes, we know that there should be a forgot password button. However due to time constraints, we have not yet implemented this feature. Since this is not a bug, but an additional feature, we will implement this in our future release of the webapp.
